lib/ansible/modules/unarchive.py

```diff
@@ -1165,7 +1165,7 @@ def main():
                 file_args['path'] = "%s/%s" % (dest, f)
                 try:
                     res_args['changed'] = module.set_fs_attributes_if_different(file_args, res_args['changed'], expand=False)
-                except (IOError, OSError) as e:
+                except OSError as e:
                     module.fail_json(msg="Unexpected error when accessing exploded file: %s" % to_native(e), **res_args)
```

Comment:
```suggestion
                    module.fail_json("Unexpected error when accessing exploded file.", exception=e, **res_args)
```

Reply:
There are more error handlers like this one in this PR which could also be updated.